python/packages/autogen-ext/pyproject.toml

```diff
@@ -50,6 +50,9 @@ video-surfer = [
 grpc = [
     "grpcio~=1.62.0", # TODO: update this once we have a stable version.
 ]
+azure-ai-inference = [
+    "azure-ai-inference>=1.0.0b6",
```

Comment:
Is there a specific reason for this version lower bound?

Reply:
No this is probably becuase of my inexperience with uv. Ill update this today. With the latest stable version.

Reply:
No this is not intentional, this is what gets added when i run `uv add "azure-ai-inference" --optional azure-ai-inference`.
Is there another command I should use? I'm not experienced with uv